src/main/resources/org/jenkinsci/plugins/workflow/job/WorkflowRun/sidepanel.jelly

```diff
@@ -28,24 +28,24 @@
     <l:side-panel>
         <l:tasks>
             <j:set var="buildUrl" value="${h.decompose(request)}"/>
-            <l:task icon="icon-up icon-md" href="${rootURL}/${it.parent.url}" title="${%Back to Project}" contextMenu="false"/>
-            <l:task icon="icon-search icon-md" href="${buildUrl.baseUrl}/" title="${%Status}" contextMenu="false"/>
-            <l:task icon="icon-notepad icon-md" href="${buildUrl.baseUrl}/changes" title="${%Changes}"/>
+            <l:task icon="symbol-up icon-md" href="${rootURL}/${it.parent.url}" title="${%Back to Project}" contextMenu="false"/>
+            <l:task icon="symbol-search icon-md" href="${buildUrl.baseUrl}/" title="${%Status}" contextMenu="false"/>
+            <l:task icon="symbol-changes icon-md" href="${buildUrl.baseUrl}/changes" title="${%Changes}"/>
             <j:choose> <!-- TODO <p:console-link/> may not currently be used as it calls getLogFile -->
                 <j:when test="${it.logText.length() > 200000}">
-                    <l:task href="${buildUrl.baseUrl}/console" icon="icon-terminal icon-md" title="${%Console Output}"/>
-                    <l:task href="${buildUrl.baseUrl}/consoleText" icon="icon-document icon-md" title="${%View as plain text}"/>
+                    <l:task href="${buildUrl.baseUrl}/console" icon="symbol-terminal icon-md" title="${%Console Output}"/>
+                    <l:task href="${buildUrl.baseUrl}/consoleText" icon="symbol-document-text icon-md" title="${%View as plain text}"/>
                 </j:when>
                 <j:otherwise>
-                    <l:task icon="icon-terminal icon-md" href="${buildUrl.baseUrl}/console" title="${%Console Output}">
-                        <l:task href="${buildUrl.baseUrl}/consoleText" icon="icon-document icon-md" title="${%View as plain text}"/>
+                    <l:task icon="symbol-terminal icon-md" href="${buildUrl.baseUrl}/console" title="${%Console Output}">
+                        <l:task href="${buildUrl.baseUrl}/consoleText" icon="symbol-document-text icon-md" title="${%View as plain text}"/>
                     </l:task>
                 </j:otherwise>
             </j:choose>
-            <l:task icon="icon-notepad icon-md" href="${buildUrl.baseUrl}/configure" title="${h.hasPermission(it,it.UPDATE)?'%Edit Build Information':'%View Build Information'}"/>
+            <l:task icon="symbol-details icon-md" href="${buildUrl.baseUrl}/configure" title="${h.hasPermission(it,it.UPDATE)?'%Edit Build Information':'%View Build Information'}"/>
```

Comment:
I'm only wondering if we should use an edit symbol or add custom one like this one
https://ionic.io/ionicons
![create-outline](https://user-images.githubusercontent.com/9051964/176562586-7c93b624-87dc-40c1-897a-5c8b02374b9e.svg)

Reply:
@jglick what do you think about this icon?

Reply:
@NotMyFault What do you think about using any of those symbols in "Edit Build Description" task:
-> edit symbol https://github.com/jenkinsci/jenkins/blob/master/war/src/main/resources/images/symbols/edit.svg
-> details symbol https://github.com/jenkinsci/jenkins/blob/master/war/src/main/resources/images/symbols/details.svg
-> custom one placed above

For me custom one fits the best here, but i do not want to introduce new symbols in one of the most used plugin, when there is no such need.

Reply:
That should probably be a gear symbol to align with the design used in other job types.

The current "Edit Build Information" icon should be the Status icon as well, that's how it's done in other job types.

Reply:
I don't agree about gear symbol. It's overused in Jenkins (also by other plugins). 
Previously it was a notepad icon, so this is another argument against it.

> The current "Edit Build Information" symbol should be made the Status symbol as well, that's how it's done in other job types.
Could you provide more details about it?

Reply:
Ok, so details symbol is big no no here (Edit Build Description), and i should also update status icon from:
symbol-search => symbol-details

Do I understand correctly?

Reply:
Yes, take a look at a freestyle job for the design in place for a better comparison to a pipeline job.

Reply:
Ok I applied those changes